### ISSUE TITLE
Why does Temml insert additional space after first open paren with nested sin()?

### ISSUE THREAD

**Issue by gwhitney** [ronkok/Temml#74]
On the demo page, if I insert `\mathrm{sign}\left(\sin(n)\right)`, the resulting mathml is
```
<math>
  <mrow>
    <mrow>
      <mtext></mtext>
      <mi>sign</mi>
    </mrow>
    <mrow>
      <mo fence="true" form="prefix">(</mo>
      <mrow>
        <mspace width="0.1667em"></mspace>
        <mi>sin</mi>
        <mo>⁡</mo>
      </mrow>
      <mo form="prefix" stretchy="false">(</mo>
      <mi>n</mi>
      <mo form="postfix" stretchy="false">)</mo>
      <mo fence="true" form="postfix">)</mo>
    </mrow>
  </mrow>
</math>
```
Why the `<mspace width="0.1667em"></mspace>` before the `<mi>sin</mi>` ? The expression looks fine to my eyes, in fact better, without it; putting the space in creates an unexpected gap in the typesetting, and there doesn't seem to be anything in the latex input that would suggest inserting space there. (I mean, it goes away if I remove the `\left` and `\right`, but I think of those as auto-sizing the parentheses, not adding space. As this is auto-generated latex code from a generator that conservatively puts in the `\left` and `\right` just in case there is something big in there, I can't "just remove the left and right" from my latex input, unfortunately.) Any way to control this insertion of space? I don't really see an option on the "Installation" page that seems relevant. Thanks.

**Comment by ronkok** [comment]
The `<mspace width="0.1667em">` is a bug. I just worked up a fix. Tomorrow, I'll write up a couple of tests and publish it. Thank you for bringing this to my attention.

That's a pity about the auto-generated `\left` and `\right`. Chrome is going to add some pretty big spaces even after the `<mspace>` bug is fixed. I don't know of any good way to reduce it.

And if I may be pedantic for a minute, I think `\operatorname` would be better semantically than `\mathrm`.

**Comment by gwhitney** [comment]
> And if I may be pedantic for a minute, I think `\operatorname` would be better semantically than `\mathrm`.

I agree. As I said, I didn't write this latex (maybe I didn't make clear I didn't even write the generator).
Looking forward to the fix!

**Comment by ronkok** [comment]
Fixed by Release v0.10.31. Thank you again for pointing out this problem.
